fenix/app/src/main/java/org/mozilla/fenix/theme/FirefoxTheme.kt

```diff
@@ -332,6 +356,23 @@ class FirefoxColors(
     // Selected tab
     var layerAccentOpaque by mutableStateOf(layerAccentOpaque)
         private set
+
+    // Status indicator dialog background
+    var layerWarning by mutableStateOf(layerWarning)
+        private set
+
+    // Status indicator dialog background
+    var layerConfirmation by mutableStateOf(layerConfirmation)
+        private set
+
+    // Status indicator dialog background
+    var layerError by mutableStateOf(layerError)
+        private set
+
+    // Status indicator dialog background
+    var layerInfo by mutableStateOf(layerInfo)
+        private set
```

Comment:
Same request as above about moving reordering these variables to match the ordering of the stickersheet.